utils/markdown/html_entities.go

```diff
@@ -1145,7 +1145,7 @@ var htmlEntities = map[string]string{
 	"in":                              "\u2208",
 	"incare":                          "\u2105",
 	"infin":                           "\u221E",
-	"infintie":                        "\u29DD",
+	"infinite":                        "\u29DD",
```

Comment:
Not sure

Reply:
False positive, take a look here: https://www.w3schools.com/charsets/ref_html_entities_i.asp and search for the infintie html entity.